packages/teleport/src/Recordings/RecordingsList.tsx

```diff
@@ -81,6 +91,21 @@ export default function RecordingsList(props: Props) {
   );
 }
 
+const iconCell = (type: RecordingType) => {
+  const iconProps = {
+    p: '1',
+    mr: '3',
+    fontSize: '3',
+  };
+  const icon = type === 'ssh' ? Icons.Cli : Icons.Desktop;
+
+  return (
+    <Cell>
+      <Icon {...iconProps} as={icon} />
```

Comment:
I think it's more readable when these attributes are close to the component itself. 
`<Icon   p={1} mr={3}  fontSize={3}..`